app/controllers/users_controller.rb

```diff
@@ -14,8 +14,25 @@ def create
     end
   end
 
+  def edit
+    @user = User.find(current_user.id)
```

Comment:
I think you can directly use `current_user` instead of `User.find(current_user.id)`